sycl/doc/design/CompilerAndRuntimeDesign.md

```diff
@@ -423,6 +423,30 @@ Case 1 can be identified in the device binary generation stage (step 1) by
 scanning the known kernels. Case 2 must be verified by the driver by checking
 for newly introduced kernels in the final link stage (step 3).
 
+#### Device Link during compilation
+
+The `-fno-sycl-rdc` flag can be used in correlation with the `-c` option
```

Comment:
```suggestion
The `-fno-sycl-rdc` flag can be used in combination with the `-c` option
```

Not sure if this is a typo, if not, you can ignore

Reply:
combination makes more sense here.